models/boleto.go

```diff
@@ -27,6 +27,7 @@ type BoletoRequest struct {
 	Buyer          Buyer           `json:"buyer"`
 	BankNumber     BankNumber      `json:"bankNumber"`
 	RequestKey     string          `json:"requestKey,omitempty"`
+	ServiceUser    string          `json:"ServiceUser,omitempty"`
```

Comment:
Opa, geralmente a TAG do JSON fica com a primeira letra minúscula, e é o padrão que está nos outros campos 😀 
```suggestion
	ServiceUser    string          `json:"serviceUser,omitempty"`
```